packages/runtime/test/integration/pool-transaction-fee-test.ts

```diff
@@ -86,4 +98,86 @@ describe("Pooled Transaction Fees Test", function () {
 			RUNTIME_ERRORS.TRANSACTION.FEES_NOT_ENOUGH
 		);
 	});
+
+	it("Should unfunded accounts be able to issue transactions of group size 3", () => {
```

Comment:
None of this accounts is unfunded

Reply:
This scenario is covered in last unit test of this file. I have renamed this unit test. Please check